.github/workflows/test.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       !contains(github.event.head_commit.message, '[ci skip]') &&
       !contains(github.event.head_commit.message, '[skip ci]')
     name: Build yagit on Linux
```

Comment:
Could you also add a cron entry to periodically run these tests, like once per week ?

Reply:
I don't see a point to checking this when there are no changes in repo. I made a change in this file because it used `ubuntu-20.04` GA runner, which was made fully unsupported exactly 3 days ago :D. `ubuntu-latest` will resolve all these problems.
https://github.com/actions/runner-images/issues/11101